pkgs/development/tools/delve/default.nix

```diff
@@ -8,14 +8,20 @@ buildGoModule rec {
     owner = "go-delve";
     repo = "delve";
     rev = "v${version}";
-    sha256 = "sha256-rW3uKf5T+ZCjZxVuSFWWXw0mhAW9Y9L83xtU98JTuik=";
+    leaveDotGit = true;  # needed for cmd/dlv/dlv_test.go:TestBuild
```

Comment:
```suggestion
```
Please disable the tests instead. I'd rather have one test less than have impurity through git.

Reply:
yeah. here and there this week i tried to fix more issues with this on darwin. it's friggin' hopeless. i'm going to disable tests again.